Code/Framework/AzCore/AzCore/IO/IOUtils.h

```diff
@@ -7,6 +7,8 @@
  */
 #pragma once
 
+#include <AzCore/IO/FileIO.h> // for OpenMode
+
```

Comment:
The OpenMode enum should be forward declared

```suggestion
namespace AZ::IO
{
    enum class OpenMode : AZ::u32;
}
```